index.html

```diff
@@ -52,6 +65,27 @@ <h3>Multi-Post Stories</h3>
         <button type="button" class="btn">See Project</button>
```

Comment:
- [ ] Please check the See Project `button` we have it double in this section.

Reply:
@CollinsTatang , I do not understand this comment, I don't see how I can take it away as this button has to appear at the end of every project.
Artur

Reply:
@CollinsTatang, I still do not see it in my project. It was not in the code when I sent it. Could it be browsers issues.